frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html

```diff
@@ -134,22 +134,34 @@ <h2>Add user</h2>
           >{{ role.value }}</mat-option
         >
       </mat-select>
-      <mat-error>You have to select a role!</mat-error> </mat-form-field
-    ><br />
-    <mat-form-field *ngIf="addUserToProjectForm.value.role !== 'manager'">
-      <mat-label>Permission</mat-label>
-      <mat-select formControlName="permission">
-        <mat-option
-          *ngFor="let permission of projectUserService.PERMISSIONS | keyvalue"
-          [value]="permission.key"
-          >{{ permission.value }}</mat-option
-        >
-      </mat-select>
-      <mat-error>You have to select a permission!</mat-error>
+      <mat-error>You have to select a role!</mat-error>
+    </mat-form-field>
+    <div *ngIf="addUserToProjectForm.value.role !== 'manager'">
+      <br />
+      <mat-form-field *ngIf="addUserToProjectForm.value.role !== 'manager'">
+        <mat-label>Permission</mat-label>
+        <mat-select formControlName="permission">
+          <mat-option
+            *ngFor="let permission of projectUserService.PERMISSIONS | keyvalue"
+            [value]="permission.key"
+            >{{ permission.value }}</mat-option
+          >
+        </mat-select>
+        <mat-error>You have to select a permission!</mat-error>
+      </mat-form-field>
+    </div>
+    <br />
+    <mat-form-field>
+      <mat-label>Reason</mat-label>
+      <input matInput formControlName="reason" />
```

Comment:
```suggestion
      <textarea matInput formControlName="reason"></textarea>
```

Reasons are usually a bit longer, it's better to be able to expand the field.

Reply:
Done